patches/0010-Remove-Tracking-Protection-Settings-from-preferences.patch

```diff
@@ -0,0 +1,25 @@
+From: Sam Macbeth <sam@cliqz.com>
+Date: Tue, 15 Sep 2020 12:17:48 +0200
+Subject: Remove Tracking Protection Settings from preferences
+
+---
+ browser/themes/shared/preferences/privacy.css | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/browser/themes/shared/preferences/privacy.css b/browser/themes/shared/preferences/privacy.css
+index e58d80c5ca..51841957a1 100644
+--- a/browser/themes/shared/preferences/privacy.css
++++ b/browser/themes/shared/preferences/privacy.css
+@@ -98,6 +98,9 @@
+ }
+ 
+ /* Content Blocking */
++#trackingGroup {
++  display: none;
++}
```

Comment:
@chrmod What do you think of this method of hiding parts of the UI? This has the advantage, compared to deleting or commenting markup, that we don't have to patch the JS part too. This change is also less likely to cause merge conflicts later.

Reply:
I like it